src/rdo.rs

```diff
@@ -833,11 +839,20 @@ pub fn rdo_mode_decision<T: Pixel>(
         modes.push(luma_mode)
       }
     });
+    } else {
+      // FIXME: please... (Just wanted to copy RAV1E_INTRA_MODES --> modes)
+      RAV1E_INTRA_MODES.iter().for_each(|&luma_mode| modes.push(luma_mode));
```

Comment:
modes.extend(RAV1E_INTRA_MODES) ? or modes.extend_with_slice, don't remember

Reply:
Tried both, but did not work and rustc suggested me "clone_from_slice()", which causes rust panic.
@lu-zero , I believe you can advise us on this?

Reply:
We can just leave it as is and I can try some things in a second pr.